tests/functional/smoke/cndump.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
+
+#doc: cndump test
+
+. common.subr
+
+kvdb_create
+
+kvs=$(kvs_create smoke-0)
+
+cmd putbin -c10 -L100 "$home" "$kvs"
+
+cmd cndump -h
+cmd cndump -hv
+
+cmd cndump cndb "$home" > "$home/tmp.cndb"
+
+cmd awk '($1=="kvset_add") {for(i=2;i<NF;i+=2){if($i=="kvsetid"){print($(i+1))}}}' < "$home/tmp.cndb" > "$home/tmp.kvset"
+cmd awk '($1=="hblock")    {print($2)}' < "$home/tmp.cndb" > "$home/tmp.hblock"
+cmd awk '($1=="kblocks")   {print($3)}' < "$home/tmp.cndb" > "$home/tmp.kblock"
+cmd awk '($1=="vblocks")   {print($3)}' < "$home/tmp.cndb" > "$home/tmp.vblock"
```

Comment:
It would't hurt to use some white space in the for loop...

Reply:
Awk scripts are supposed to be hard to read :)

I just learned you don't need parens around awk patterns.... but that's how I've done it since, oh since the 1980s.  It will be a major challenge to change that habit.